src/twisted/newsfragments/9125.misc

```diff
@@ -0,0 +1 @@
+Improved the exception text for mutually exclusive function arguments.
```

Comment:
misc fragments should be empty.  their text won't be included in a news file, it'll just be dropped.

if the text were to be kept and included, the fragment should follow the style guidelines from the twisted wiki.  roughly, "twisted.internet.ssl.CertificateOptions now raises an exception with better text when passed mutually exclusive arguments."

Reply:
And in this case, the change is actually an improvement to `twisted.python.deprecate`, so making the fragment text about the API there probably makes sense too.  (CertificateOptions could be given as an example beneficiary.)

Reply:
Ah, I see. Do you think it's worth replacing this newsfragment with an empty one, even though it'll make no difference in the long run (once the newsfragment is deleted after the next release)? Or change it to a non-`.misc` newsfragment? Or should I merge it as-is?

Reply:
I don't think it will hurt to merge it as-is and then you don't have to wait for another round of CI.

Reply:
Good point. Merged. :)